tic-tac-toe-app/App.jsx

```diff
@@ -1,5 +1,87 @@
+function Cells(props) {
```

Comment:
There are four components here in App.jsx. In order to organize your code in a more modular way, I'd recommend by separating out this file by component (ie `<Cells/>` could go into a "Cells.jsx" file and `<Board/>` could go into a "Board.jsx" file, etc).  By having logical chunks of code, this app will become more accessible and easier for other developers to reason about.